suites/test_bulk_operations.py

```diff
@@ -28,7 +28,7 @@ def modules_bulk_edit_metadata(self):
         utilities.page_reload(self.driver)
         search_beta_page.select_repo(self.driver, fixture.repo_name)
         search_beta_page.search_titles(self.driver, constants.bulk_operations_modules)
-        poll(lambda: len(utilities.find_elements_by_css_selector(self.driver, locators.MODULES_LIST_CSS))== 4,
+        poll(lambda: len(utilities.find_elements_by_css_selector(self.driver, locators.MODULES_LIST_CSS))== 5,
```

Comment:
I have added a comment here https://gitlab.cee.redhat.com/ccs-tools/uploader-sample-repo-pantheon2/-/merge_requests/11/diffs

The search result here should return 4 values only and all the changes you have made to change to count from 4 to 5 and 3 to 4 should not be required

Reply:
as uploader repo content is changed reverting all changes.